### PR TITLE
UI: Remove UNUSED_PARAMETER where unnecessary

### PR DESCRIPTION
**Pull request by cg2121**
### Description
Since cpp allows removing the unused parameters from the function name, UNUSED_PARAMETER is not needed, unless it is in an ifdef.

### Motivation and Context
Less code

### How Has This Been Tested?
Compiled OBS to make sure there were no warnings.

### Types of changes
- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by tytan652** [comment]
If the functions is not from a class (so no declaration). You should put the parameter name as comment. Like it was done in AJA:

- https://github.com/obsproject/obs-studio/blob/8befd830a9807e35762006b9c715cdb133ab2c75/UI/frontend-plugins/aja-output-ui/aja-ui-main.cpp#L453
- https://github.com/obsproject/obs-studio/pull/6771#discussion_r925232452

File: UI/frontend-plugins/decklink-captions/decklink-captions.cpp [hunk 1]
```diff
@@ -67,11 +67,9 @@ void DecklinkCaptionsUI::on_source_currentIndexChanged(int)
 	captions->start();
 }
 
-static void caption_callback(void *param, obs_source_t *source,
+static void caption_callback(void *, obs_source_t *,
 			     const struct obs_source_cea_708 *captions)
 {
-	UNUSED_PARAMETER(param);
-	UNUSED_PARAMETER(source);
```
Comment (tytan652) [inline UI/frontend-plugins/decklink-captions/decklink-captions.cpp:74]: If the functions is not from a class (so no declaration). You should put the parameter name as comment. Like it was done in AJA:

- https://github.com/obsproject/obs-studio/blob/8befd830a9807e35762006b9c715cdb133ab2c75/UI/frontend-plugins/aja-output-ui/aja-ui-main.cpp#L453
- Where it was discussed: https://github.com/obsproject/obs-studio/pull/6771#discussion_r925232452

Edit: The purpose is mostly to not lose parameters definition/name and their meaning.

File: UI/frontend-plugins/decklink-output-ui/decklink-ui-main.cpp [hunk 2]
```diff
@@ -114,10 +114,8 @@ OBSData load_preview_settings()
 void on_preview_scene_changed(enum obs_frontend_event event, void *param);
 void render_preview_source(void *param, uint32_t cx, uint32_t cy);
 
-static void preview_tick(void *param, float sec)
+static void preview_tick(void *param, float)
 {
-	UNUSED_PARAMETER(sec);
-
```
Comment (tytan652) [inline UI/frontend-plugins/decklink-output-ui/decklink-ui-main.cpp:120]: Same

File: UI/obs-app.cpp [hunk 3]
```diff
@@ -2494,7 +2491,7 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 	"Woops, OBS has crashed!\n\nWould you like to copy the crash log " \
 	"to the clipboard? The crash log will still be saved to:\n\n%s"
 
-static void main_crash_handler(const char *format, va_list args, void *param)
+static void main_crash_handler(const char *format, va_list args, void *)
```
Comment (tytan652) [inline UI/obs-app.cpp:2494]: Same

File: UI/window-basic-main-outputs.cpp [hunk 4]
```diff
@@ -47,14 +47,12 @@ static void OBSStreamStopping(void *data, calldata_t *params)
...
 
-static void OBSStartStreaming(void *data, calldata_t *params)
+static void OBSStartStreaming(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 	output->streamingActive = true;
 	os_atomic_set_bool(&streaming_active, true);
 	QMetaObject::invokeMethod(output->main, "StreamingStart");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:57]: Same

File: UI/window-basic-main-outputs.cpp [hunk 5]
```diff
@@ -73,15 +71,13 @@ static void OBSStopStreaming(void *data, calldata_t *params)
...
-static void OBSStartRecording(void *data, calldata_t *params)
+static void OBSStartRecording(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 
 	output->recordingActive = true;
 	os_atomic_set_bool(&recording_active, true);
 	QMetaObject::invokeMethod(output->main, "RecordingStart");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:84]: Same

File: UI/window-basic-main-outputs.cpp [hunk 6]
```diff
@@ -100,12 +96,10 @@ static void OBSStopRecording(void *data, calldata_t *params)
 				  Q_ARG(QString, arg_last_error));
 }
 
-static void OBSRecordStopping(void *data, calldata_t *params)
+static void OBSRecordStopping(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 	QMetaObject::invokeMethod(output->main, "RecordStopping");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:108]: Same

File: UI/window-basic-main-outputs.cpp [hunk 7]
```diff
@@ -122,15 +116,13 @@ static void OBSRecordFileChanged(void *data, calldata_t *params)
...
-static void OBSStartReplayBuffer(void *data, calldata_t *params)
+static void OBSStartReplayBuffer(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 
 	output->replayBufferActive = true;
 	os_atomic_set_bool(&replaybuf_active, true);
 	QMetaObject::invokeMethod(output->main, "ReplayBufferStart");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:133]: Same

File: UI/window-basic-main-outputs.cpp [hunk 8]
```diff
@@ -144,12 +136,10 @@ static void OBSStopReplayBuffer(void *data, calldata_t *params)
 				  Q_ARG(int, code));
 }
 
-static void OBSReplayBufferStopping(void *data, calldata_t *params)
+static void OBSReplayBufferStopping(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 	QMetaObject::invokeMethod(output->main, "ReplayBufferStopping");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:152]: Same

File: UI/window-basic-main-outputs.cpp [hunk 9]
```diff
@@ -159,15 +149,13 @@ static void OBSReplayBufferSaved(void *data, calldata_t *)
...
-static void OBSStartVirtualCam(void *data, calldata_t *params)
+static void OBSStartVirtualCam(void *data, calldata_t *)
 {
 	BasicOutputHandler *output = static_cast<BasicOutputHandler *>(data);
 
 	output->virtualCamActive = true;
 	os_atomic_set_bool(&virtualcam_active, true);
 	QMetaObject::invokeMethod(output->main, "OnVirtualCamStart");
-
-	UNUSED_PARAMETER(params);
```
Comment (tytan652) [inline UI/window-basic-main-outputs.cpp:170]: Same

File: UI/window-basic-main.cpp [hunk 10]
```diff
@@ -2973,7 +2973,7 @@ void OBSBasic::RemoveScene(OBSSource source)
 		api->on_event(OBS_FRONTEND_EVENT_SCENE_LIST_CHANGED);
 }
 
-static bool select_one(obs_scene_t *scene, obs_sceneitem_t *item, void *param)
+static bool select_one(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-main.cpp:2976]: Same

File: UI/window-basic-main.cpp [hunk 11]
```diff
@@ -8124,7 +8104,7 @@ void OBSBasic::on_actionPasteTransform_triggered()
 		undo_redo, undo_redo, undo_data, redo_data);
 }
 
-static bool reset_tr(obs_scene_t *scene, obs_sceneitem_t *item, void *param)
+static bool reset_tr(obs_scene_t *, obs_sceneitem_t *item, void *)
 {
```
Comment (tytan652) [inline UI/window-basic-main.cpp:8108]: Same

File: UI/window-basic-main.cpp [hunk 12]
```diff
@@ -8216,7 +8194,7 @@ static void SetItemTL(obs_sceneitem_t *item, const vec3 &tl)
 	obs_sceneitem_set_pos(item, &pos);
 }
 
-static bool RotateSelectedSources(obs_scene_t *scene, obs_sceneitem_t *item,
+static bool RotateSelectedSources(obs_scene_t *, obs_sceneitem_t *item,
```
Comment (tytan652) [inline UI/window-basic-main.cpp:8197]: Same

File: UI/window-basic-main.cpp [hunk 13]
```diff
@@ -8297,7 +8274,7 @@ void OBSBasic::on_actionRotate180_triggered()
 			  undo_redo, undo_redo, undo_data, redo_data);
 }
 
-static bool MultiplySelectedItemScale(obs_scene_t *scene, obs_sceneitem_t *item,
+static bool MultiplySelectedItemScale(obs_scene_t *, obs_sceneitem_t *item,
 				      void *param)
```
Comment (tytan652) [inline UI/window-basic-main.cpp:8278]: Same

File: UI/window-basic-main.cpp [hunk 14]
```diff
@@ -8363,7 +8339,7 @@ void OBSBasic::on_actionFlipVertical_triggered()
 			  undo_redo, undo_redo, undo_data, redo_data);
 }
 
-static bool CenterAlignSelectedItems(obs_scene_t *scene, obs_sceneitem_t *item,
+static bool CenterAlignSelectedItems(obs_scene_t *, obs_sceneitem_t *item,
```
Comment (tytan652) [inline UI/window-basic-main.cpp:8342]: Same

File: UI/window-basic-preview.cpp [hunk 15]
```diff
@@ -110,8 +110,7 @@ static bool CloseFloat(float a, float b, float epsilon = 0.01)
 	return abs(a - b) <= epsilon;
 }
 
-static bool FindItemAtPos(obs_scene_t *scene, obs_sceneitem_t *item,
-			  void *param)
+static bool FindItemAtPos(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:113]: Same

File: UI/window-basic-preview.cpp [hunk 16]
```diff
@@ -237,8 +235,7 @@ OBSSceneItem OBSBasicPreview::GetItemAtPos(const vec2 &pos, bool selectBelow)
 	return data.item;
 }
 
-static bool CheckItemSelected(obs_scene_t *scene, obs_sceneitem_t *item,
-			      void *param)
+static bool CheckItemSelected(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:238]: Same

File: UI/window-basic-preview.cpp [hunk 17]
```diff
@@ -334,8 +330,7 @@ struct HandleFindData {
 	}
 };
 
-static bool FindHandleAtPos(obs_scene_t *scene, obs_sceneitem_t *item,
-			    void *param)
+static bool FindHandleAtPos(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:333]: Same

File: UI/window-basic-preview.cpp [hunk 18]
```diff
@@ -670,7 +664,7 @@ void OBSBasicPreview::UpdateCursor(uint32_t &flags)
 		setCursor(Qt::OpenHandCursor);
 }
 
-static bool select_one(obs_scene_t *scene, obs_sceneitem_t *item, void *param)
+static bool select_one(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:667]: Same

File: UI/window-basic-preview.cpp [hunk 19]
```diff
@@ -815,8 +808,7 @@ struct SelectedItemBounds {
 	vec3 tl, br;
 };
 
-static bool AddItemBounds(obs_scene_t *scene, obs_sceneitem_t *item,
-			  void *param)
+static bool AddItemBounds(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:811]: Same

File: UI/window-basic-preview.cpp [hunk 20]
```diff
@@ -875,7 +866,7 @@ struct OffsetData {
 	vec3 tl, br, offset;
 };
 
-static bool GetSourceSnapOffset(obs_scene_t *scene, obs_sceneitem_t *item,
+static bool GetSourceSnapOffset(obs_scene_t *, obs_sceneitem_t *item,
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:869]: Same

File: UI/window-basic-preview.cpp [hunk 21]
```diff
@@ -978,7 +968,7 @@ void OBSBasicPreview::SnapItemMovement(vec2 &offset)
 	}
 }
 
-static bool move_items(obs_scene_t *scene, obs_sceneitem_t *item, void *param)
+static bool move_items(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:971]: Same

File: UI/window-basic-preview.cpp [hunk 22]
```diff
@@ -1095,20 +1084,18 @@ static bool IntersectBox(matrix4 transform, float x1, float x2, float y1,
...
 	if (obs_sceneitem_selected(item))
 		data->sceneItems.push_back(item);
 
-	UNUSED_PARAMETER(scene);
 	return true;
 }
 
-static bool FindItemsInBox(obs_scene_t *scene, obs_sceneitem_t *item,
-			   void *param)
+static bool FindItemsInBox(obs_scene_t *, obs_sceneitem_t *item, void *param)
```
Comment (tytan652) [inline UI/window-basic-preview.cpp:1098]: Same

**Review by tytan652 — CHANGES_REQUESTED** [review]
This review only covers static function, if a non-static has the same "meaning loss" issue, the same should be applied.

File: UI/frontend-plugins/decklink-captions/decklink-captions.cpp (hunk 1 above)
Comment (cg2121) [reply UI/frontend-plugins/decklink-captions/decklink-captions.cpp]: Fixed

File: UI/frontend-plugins/decklink-output-ui/decklink-ui-main.cpp (hunk 2 above)
Comment (cg2121) [reply UI/frontend-plugins/decklink-output-ui/decklink-ui-main.cpp]: Fixed

File: UI/obs-app.cpp (hunk 3 above)
Comment (cg2121) [reply UI/obs-app.cpp]: Fixed

File: UI/window-basic-main-outputs.cpp (hunk 4 above)
Comment (cg2121) [reply UI/window-basic-main-outputs.cpp]: Fixed

File: UI/window-basic-main-outputs.cpp (hunk 5 above)
Comment (cg2121) [reply UI/window-basic-main-outputs.cpp]: Fixed

File: UI/window-basic-preview.cpp (hunk 18 above)
Comment (cg2121) [reply UI/window-basic-preview.cpp]: Fixed

File: UI/window-basic-preview.cpp (hunk 19 above)
Comment (cg2121) [reply UI/window-basic-preview.cpp]: Fixed

File: UI/window-basic-preview.cpp (hunk 20 above)
Comment (cg2121) [reply UI/window-basic-preview.cpp]: Fixed

File: UI/window-basic-preview.cpp (hunk 21 above)
Comment (cg2121) [reply UI/window-basic-preview.cpp]: Fixed

File: UI/window-basic-preview.cpp (hunk 22 above)
Comment (cg2121) [reply UI/window-basic-preview.cpp]: Fixed

File: UI/window-basic-main.cpp [hunk 23]
```diff
@@ -3751,7 +3751,7 @@ void OBSBasic::TimedCheckForUpdates()
 #endif
 }
 
-void OBSBasic::CheckForUpdates(bool manualUpdate)
+void OBSBasic::CheckForUpdates(bool)
```
Comment (tytan652) [inline UI/window-basic-main.cpp:3754]: `manualUpdate` is used in a `#ifdef`

File: UI/window-basic-main.cpp [hunk 24]
```diff
@@ -3765,8 +3765,6 @@ void OBSBasic::CheckForUpdates(bool manualUpdate)
 	updateCheckThread.reset(new AutoUpdateThread(manualUpdate));
 	updateCheckThread->start();
 #endif
-
-	UNUSED_PARAMETER(manualUpdate);
```
Comment (tytan652) [inline UI/window-basic-main.cpp:3769]: `manualUpdate` is used in a `#ifdef`

```
#ifndef _WIN32
UNUSED_PARAMETER(manualUpdate)
#endif
```

Comment (cg2121) [reply UI/window-basic-main.cpp:3769]: Fixed

Comment (tytan652) [reply UI/window-basic-main.cpp:3769]: The separate `#ifndef` was to cover Linux and macOS at the same time.

**Review by tytan652 — APPROVED** [review]
LGTM

File: UI/properties-view.cpp [hunk 25]
```diff
@@ -1910,16 +1910,9 @@ void WidgetInfo::GroupChanged(const char *setting)
 						  : true);
 }
 
-void WidgetInfo::EditListReordered(const QModelIndex &parent, int start,
-				   int end, const QModelIndex &destination,
-				   int row)
+void WidgetInfo::EditListReordered(const QModelIndex &, int, int,
+				   const QModelIndex &, int)
```
Comment (gxalpha) [inline UI/properties-view.cpp:1914]: Arguments can be removed

File: UI/window-basic-filters.cpp [hunk 26]
```diff
@@ -1169,18 +1165,16 @@ void OBSBasicFilters::FilterNameEdited(QWidget *editor, QListWidget *list)
 	editActive = false;
 }
 
-void OBSBasicFilters::AsyncFilterNameEdited(
-	QWidget *editor, QAbstractItemDelegate::EndEditHint endHint)
+void OBSBasicFilters::AsyncFilterNameEdited(QWidget *editor,
+					    QAbstractItemDelegate::EndEditHint)
```
Comment (gxalpha) [inline UI/window-basic-filters.cpp:1169]: `QAbstractItemDelegate::EndEditHint` can be removed

File: UI/window-basic-filters.cpp [hunk 27]
```diff
@@ -1169,18 +1165,16 @@ void OBSBasicFilters::FilterNameEdited(QWidget *editor, QListWidget *list)
...
 {
 	FilterNameEdited(editor, ui->asyncFilters);
-	UNUSED_PARAMETER(endHint);
 }
 
-void OBSBasicFilters::EffectFilterNameEdited(
-	QWidget *editor, QAbstractItemDelegate::EndEditHint endHint)
+void OBSBasicFilters::EffectFilterNameEdited(QWidget *editor,
+					     QAbstractItemDelegate::EndEditHint)
```
Comment (gxalpha) [inline UI/window-basic-filters.cpp:1175]: `QAbstractItemDelegate::EndEditHint` can be removed

File: UI/window-basic-main-transitions.cpp [hunk 28]
```diff
@@ -627,13 +625,11 @@ void OBSBasic::on_transitionProps_clicked()
 	menu.exec(QCursor::pos());
 }
 
-void OBSBasic::on_transitionDuration_valueChanged(int value)
+void OBSBasic::on_transitionDuration_valueChanged(int)
```
Comment (gxalpha) [inline UI/window-basic-main-transitions.cpp:628]: Parameter can be removed

File: UI/window-basic-main-transitions.cpp [hunk 29]
```diff
@@ -952,13 +948,11 @@ int OBSBasic::GetTbarPosition()
 	return tBar->value();
 }
 
-void OBSBasic::on_tbar_position_valueChanged(int value)
+void OBSBasic::on_tbar_position_valueChanged(int)
```
Comment (gxalpha) [inline UI/window-basic-main-transitions.cpp:951]: Parameter can be removed

File: UI/window-basic-main.cpp [hunk 30]
```diff
@@ -7863,11 +7852,9 @@ QModelIndexList OBSBasic::GetAllSelectedSourceItems()
 	return ui->sources->selectionModel()->selectedIndexes();
 }
 
-void OBSBasic::on_preview_customContextMenuRequested(const QPoint &pos)
+void OBSBasic::on_preview_customContextMenuRequested(const QPoint &)
```
Comment (gxalpha) [inline UI/window-basic-main.cpp:7855]: Parameter can be removed

File: UI/window-basic-main.cpp [hunk 31]
```diff
@@ -6359,7 +6350,7 @@ static void RenameListItem(OBSBasic *parent, QListWidget *listWidget,
 }
 
 void OBSBasic::SceneNameEdited(QWidget *editor,
-			       QAbstractItemDelegate::EndEditHint endHint)
+			       QAbstractItemDelegate::EndEditHint)
```
Comment (gxalpha) [inline UI/window-basic-main.cpp:6353]: Parameter can be removed

File: UI/window-basic-main.cpp [hunk 32]
```diff
@@ -7893,7 +7880,7 @@ void OBSBasic::ProgramViewContextMenuRequested(const QPoint &)
 	popup.exec(QCursor::pos());
 }
 
-void OBSBasic::PreviewDisabledMenu(const QPoint &pos)
+void OBSBasic::PreviewDisabledMenu(const QPoint &)
```
Comment (gxalpha) [inline UI/window-basic-main.cpp:7883]: Parameter can be removed

File: UI/window-basic-settings.cpp [hunk 33]
```diff
@@ -4051,7 +4051,7 @@ void OBSBasicSettings::on_advOutFFPathBrowse_clicked()
 	ui->advOutFFRecPath->setText(dir);
 }
 
-void OBSBasicSettings::on_advOutEncoder_currentIndexChanged(int idx)
+void OBSBasicSettings::on_advOutEncoder_currentIndexChanged(int)
```
Comment (gxalpha) [inline UI/window-basic-settings.cpp:4054]: Parameter can be removed

File: UI/window-missing-files.cpp [hunk 34]
```diff
@@ -49,7 +49,7 @@ MissingFilesPathItemDelegate::MissingFilesPathItemDelegate(
 
 QWidget *MissingFilesPathItemDelegate::createEditor(
 	QWidget *parent, const QStyleOptionViewItem & /* option */,
-	const QModelIndex &index) const
+	const QModelIndex &) const
```
Comment (gxalpha) [inline UI/window-missing-files.cpp:52]: `QStyleOptionViewItem` and `QModelIndex` can be removed

**Review by sebastian-s-beckmann — COMMENTED** [review]
I hope I didn't miss any, so here's the general gist:
Qt Slots can have a shorter signature than the signal connected to them, any additional parameters will just be ignored. In the cases where we're removing any names, it makes sense to just make the signature shorter and remove the parameters altogether. If someone needs them in the future, they'd have to look them up anyways (since we removed the names).

I marked all instances that I could immediately identify as slots, if there are any others that are touched by this PR, feel free to change them as well. If some aren't actually slots, ignore them, I apologize.

File: UI/window-missing-files.cpp (hunk 34 above)
Comment (cg2121) [reply UI/window-missing-files.cpp:52]: This only works for slots, and this is not a slot.

File: UI/window-basic-main.cpp (hunk 32 above)
Comment (cg2121) [reply UI/window-basic-main.cpp:7883]: Only works with auto-generated slots.

File: UI/window-basic-main-transitions.cpp (hunk 29 above)
Comment (cg2121) [reply UI/window-basic-main-transitions.cpp:951]: Function was removed.

File: UI/window-basic-filters.cpp (hunk 26 above)
Comment (cg2121) [reply UI/window-basic-filters.cpp:1169]: Doesn't work.

File: UI/window-basic-filters.cpp (hunk 27 above)
Comment (cg2121) [reply UI/window-basic-filters.cpp:1175]: Doesn't work. As removing parameters only works with auto generated slots.

File: UI/properties-view.cpp (hunk 25 above)
Comment (cg2121) [reply UI/properties-view.cpp:1914]: Needs to be a auto generated slot.

File: UI/window-basic-main.cpp (hunk 31 above)
Comment (cg2121) [reply UI/window-basic-main.cpp:6353]: Needs to be a auto generated slot.

File: UI/window-basic-main-transitions.cpp (hunk 28 above)
Comment (cg2121) [reply UI/window-basic-main-transitions.cpp]: Fixed

File: UI/window-basic-main.cpp (hunk 30 above)
Comment (cg2121) [reply UI/window-basic-main.cpp]: Fixed

File: UI/window-basic-settings.cpp (hunk 33 above)
Comment (cg2121) [reply UI/window-basic-settings.cpp]: Fixed

File: UI/window-basic-main-outputs.cpp (hunk 6 above)
Comment (cg2121) [reply UI/window-basic-main-outputs.cpp]: Fixed

File: UI/window-basic-filters.cpp (hunk 26 above)
Comment (gxalpha) [reply UI/window-basic-filters.cpp:1170]: Sure works for me.
Can't say why exactly it doesn't work for you without knowing the error, but make sure to also change the arguments used in the `SIGNAL` macro.

File: UI/window-basic-filters.cpp (hunk 27 above)
Comment (gxalpha) [reply UI/window-basic-filters.cpp:1176]: See above.

File: UI/window-basic-main.cpp (hunk 31 above)
Comment (gxalpha) [reply UI/window-basic-main.cpp:6443]: See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

File: UI/window-basic-main.cpp (hunk 32 above)
Comment (gxalpha) [reply UI/window-basic-main.cpp:7984]: See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

File: UI/properties-view.cpp (hunk 25 above)
Comment (gxalpha) [reply UI/properties-view.cpp:1918]: See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

Comment (cg2121) [reply UI/properties-view.cpp]: Fixed

File: UI/window-basic-filters.cpp (hunk 26 above)
Comment (cg2121) [reply UI/window-basic-filters.cpp]: Fixed

File: UI/window-basic-filters.cpp (hunk 27 above)
Comment (cg2121) [reply UI/window-basic-filters.cpp]: Fixed

File: UI/window-basic-main.cpp (hunk 31 above)
Comment (cg2121) [reply UI/window-basic-main.cpp]: Fixed

File: UI/window-basic-main.cpp (hunk 32 above)
Comment (cg2121) [reply UI/window-basic-main.cpp]: Fixed. Only seems to work for me when I convert it to an auto-generated slot.

Comment (gxalpha) [reply UI/window-basic-main.cpp]: That's probably the better solution anyways.

**Pull request merged**